demo/trackLoader.js

```diff
@@ -250,6 +279,7 @@ async function loadTracksCallbackHelper (s3, bucket, name, trackShaderMaterial,
 
     const normalTrackColor = trackShaderMaterial.uniforms.color.value.getHex();
     const propagatedTrackColor = 0x55AAFF;
+    const startOrEndColor = 0xFF55AA;
```

Comment:
Make separate colors for start and end. (Which can be the same color for now. But you have separate state bits. Might as well reflect that here.)